examples/cmake/FindGeodeNative.cmake.in

```diff
@@ -114,12 +113,12 @@ if (GeodeNative_FIND_COMPONENTS)
 endif()
 
 include(FindPackageHandleStandardArgs)
-FIND_PACKAGE_HANDLE_STANDARD_ARGS(GeodeNative
+FIND_PACKAGE_HANDLE_STANDARD_ARGS(@PRODUCT_NAME_NOSPACE@
                                   REQUIRED_VARS ${_GEODE_NATIVE_REQUIRED_VARS}
                                   VERSION_VAR GEODE_NATIVE_VERSION_STRING)
 
 # Copy the results to the output variables and target.
-if(GeodeNative_FOUND)
+if(@PRODUCT_NAME_NOSPACE@_FOUND)
   if(NOT TARGET ${GEODE_NATIVE_CPP_TARGET})
     set(GEODE_NATIVE_CPP_TARGET "GeodeNative::cpp")
```

Comment:
Would we not want to parameterize this target? It should match all the other variable names and product.